src/spring/azext_spring/commands.py

```diff
@@ -92,6 +95,21 @@ def load_command_table(self, _):
         client_factory=cf_spring_20220101preview
     )
 
+    application_accelerator_cmd_group = CliCommandType(
+        operations_tmpl='azext_spring.application_accelerator#{}',
+        client_factory=cf_spring_20221101preview
+    )
+
+    predefined_accelerator_cmd_group = CliCommandType(
+        operations_tmpl='azext_spring.predefined_accelerator#{}',
+        client_factory=cf_spring_20221101preview
+    )
+
+    customized_accelerator_cmd_group = CliCommandType(
+        operations_tmpl='azext_spring.customized_accelerator#{}',
+        client_factory=cf_spring_20221101preview
+    )
```

Comment:
This command group is used to specify API version. I encourgage to use the same API version for acc feature. Here you can only keep L98-L101